### PR TITLE
Migrated syntax to Swift 3 - pull for correct branch

### PR DESCRIPTION
**Pull request by YoniTsafir**
See #220

### CODE REVIEW

**Comment by YoniTsafir** [comment]
I agree with all you comments in #220 BTW @JakeLin, won't get to fixing them soon though so go ahead :)

File: IBAnimatable.playground/Pages/Predefined Animations.xcplaygroundpage/Contents.swift [hunk 1]
```diff
@@ -34,6 +34,9 @@ view.y = 200
 
 // Uncomment one line to play the animation
 
+//view.moveX()
+//view.moveY()
+//view.moveXY()
```
Comment (JakeLin) [inline IBAnimatable.playground/Pages/Predefined Animations.xcplaygroundpage/Contents.swift:39]: That should be gone. I fix it later.

**Comment by JakeLin** [comment]
@theyonibomber thanks, you are in 😉. I will set up the correct CI for this branch. Please let me know if you have any question.

**Comment by JakeLin** [comment]
Close #220 and related to #221

**Pull request merged**